api/tslint.json

```diff
@@ -6,6 +6,8 @@
     "jsRules": {},
     "rules": {
         "object-literal-key-quotes": false,
+        "object-literal-sort-keys": false,
```

Comment:
Sort keys is generally a good idea, why is it disabled?

Reply:
When keys are sorted, you're not able to keep keys that have a relation to each other close to each other. For example firstName and lastName.

Reply:
Reverted this change, so that tslint checks for the sort order of keys again.